src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/TimeoutsTest.java

```diff
@@ -164,11 +172,28 @@ public void testTimeoutsConfigured() throws Exception {
         final DatabaseEngine de = DatabaseFactory.getConnection(this.dbProps);
 
         int loginTimeoutInSeconds = DriverManager.getLoginTimeout();
-        int connectionTimeoutInMs = de.getConnection().getNetworkTimeout();
+        int socketConnectionTimeoutInMs = de.getConnection().getNetworkTimeout();
+        int checkConnectionTimeoutInMs = de.getProperties().getCheckConnectionTimeout();
 
         assertEquals("Is the login timeout of the DB connection the expected?", LOGIN_TIMEOUT_SECONDS, loginTimeoutInSeconds);
         assertEquals("Is the socket timeout of the DB connection the expected?",
-                TimeUnit.SECONDS.toMillis(SOCKET_TIMEOUT_SECONDS), connectionTimeoutInMs);
+                TimeUnit.SECONDS.toMillis(SOCKET_TIMEOUT_SECONDS), socketConnectionTimeoutInMs);
+        assertEquals("Is the check connection timeout of the DB connection the expected?",
+                TimeUnit.SECONDS.toSeconds(CHECK_CONNECTION_TIMEOUT_SECONDS), checkConnectionTimeoutInMs);
```

Comment:
Connection timeout config is PDB stuff, it's not "in milliseconds", so `checkConnectionTimeoutInMs` should be named accordingly.
Also, as the name implies, CHECK_CONNECTION_TIMEOUT_SECONDS is already seconds, you don't need to convert seconds to seconds :smile: 
```suggestion
        assertEquals("Is the check connection timeout of the DB connection the expected?",
                CHECK_CONNECTION_TIMEOUT_SECONDS, checkConnectionTimeout);
```